src/API.jl

```diff
@@ -780,6 +780,7 @@ function activate()
     Base.ACTIVE_PROJECT[] = nothing
     p = Base.active_project()
     p === nothing || printpkgstyle(Context(), :Activating, "environment at $(pathrepr(p))")
+    add_snapshot_to_undo()
```

Comment:
This could perhaps be done in `print_env` instead.

Reply:
It is done there as well for the modified state but you want a "base state" as well. I know what you think and I have tried it but it became messy.